### PR TITLE
Fix composer.json

### PR DESCRIPTION
**Pull request by thewilkybarkid**
These are a few commits that fix some issues in the project's `composer.json`.

I've separated them out in case you don't want some of them; I can squash any/all if needed.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,9 +1,10 @@
 {
     "name": "sensiolabs/security-advisories",
-    "require": {
+    "description": "Database of known security vulnerabilities in various PHP projects and libraries",
+    "require-dev": {
```
Comment (GrahamCampbell) [inline composer.json]: ?

Comment (thewilkybarkid) [inline composer.json]: `symfony/yaml` is only needed by the validator.

Comment (GrahamCampbell) [inline composer.json]: ah, right :)

**Comment by stof** [comment]
Why is the lock file removed ?

**Pull request merged**